config-path.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const userHome = require('user-home');
+
+const env = process.env;
+const name = 'js-v8flags';
+
+function macos () {
+	const library = path.join(userHome, 'Library');
+	return path.join(library, 'Caches', name);
+}
+
+function windows () {
+	const appData = env.LOCALAPPDATA || path.join(userHome, 'AppData', 'Local');
+	return path.join(appData, name, 'Cache');
+}
+
+// https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+function linux () {
+	const username = path.basename(userHome);
+	return path.join(env.XDG_CACHE_HOME || path.join(userHome, '.cache'), name);
+}
+
+if (process.platform === 'darwin') {
```

Comment:
Alright after giving this more debugging time I'm still stuck. What do you think of reversing the earlier change and returning a function instead passing it he platform instead of using a global?
Or dropping 0.10 support but you probably have a reason not to ...?